src/frontend/Komponenter/Personoversikt/Dokumenter.tsx

```diff
@@ -49,37 +57,75 @@ const Dokumenter: React.FC<{ personopplysninger: IPersonopplysninger }> = ({
-                                                                onClick={() =>
-                                                                    hentDokument(dokument)
-                                                                }
-                                                                href={'#'}
-                                                            >
-                                                                {dokument.tittel}
-                                                            </Lenke>
-                                                        </Td>
-                                                    </tr>
+                                        {Object.keys(grupperteDokumenter).map(
+                                            (journalpostId: string) => {
+                                                return grupperteDokumenter[journalpostId].map(
+                                                    (dokument: Dokumentinfo, indeks: number) => {
+                                                        if (indeks === 0) {
+                                                            return (
+                                                                <TrHoveddokument key={indeks}>
+                                                                    <Td>
+                                                                        {formaterNullableIsoDatoTid(
+                                                                            dokument.dato
+                                                                        )}
+                                                                    </Td>
+                                                                    <Td>
+                                                                        <Lenke
+                                                                            onClick={() =>
+                                                                                hentDokument(
+                                                                                    dokument
+                                                                                )
+                                                                            }
+                                                                            href={'#'}
+                                                                        >
+                                                                            {dokument.tittel}
+                                                                        </Lenke>
+                                                                    </Td>
+                                                                    <Td>
+                                                                        <Normaltekst>
+                                                                            {dokument.journalstatus}
```

Comment:
Er dette en enum-verdi eller tekst? Hvis enum - trenger vi kanskje en `journalstatusTilTekst`-funksjon for å oversette og unngå "CAPS_LOCK_TEKST" :)

Reply:
Yes, la til det i eb68268 :)